Controller/MediaAdminController.php

```diff
@@ -42,22 +41,20 @@ public function createAction(Request $request = null)
     /**
      * {@inheritdoc}
      */
-    public function render($view, array $parameters = array(), Response $response = null, Request $request = null)
+    public function render($view, array $parameters = array(), Response $response = null)
```

Comment:
http://symfony.com/doc/current/contributing/code/bc.html

@core23 @greg0ire are we following Symfony BC promise?

If yes, then this is BC, because removing the last parameter is BC on symfony.